pkg/api/vlabs/types.go

```diff
@@ -58,6 +58,7 @@ type AzProfile struct {
 type FeatureFlags struct {
 	EnableCSERunInBackground bool `json:"enableCSERunInBackground,omitempty"`
 	BlockOutboundInternet    bool `json:"blockOutboundInternet,omitempty"`
+	DockerEngine             bool `json:"dockerEngine,omitempty"`
```

Comment:
Requesting a more descriptive FeatureFlag name here

Reply:
Preference for just `DockerEngine`. The other two have verbs because they change the way various workflows operate. Whereas the verb for `DockerEngine` is just something superfluous like "use docker-engine".